common/src/main/java/com/wynntils/features/user/WorldWaypointDistanceFeature.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.mojang.blaze3d.platform.Window;
+import com.mojang.math.Matrix4f;
+import com.mojang.math.Vector3d;
+import com.mojang.math.Vector3f;
+import com.mojang.math.Vector4f;
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.managers.Model;
+import com.wynntils.gui.render.FontRenderer;
+import com.wynntils.gui.render.HorizontalAlignment;
+import com.wynntils.gui.render.RenderUtils;
+import com.wynntils.gui.render.VerticalAlignment;
+import com.wynntils.mc.event.RenderEvent;
+import com.wynntils.mc.event.RenderLevelEvent;
+import com.wynntils.mc.objects.CommonColors;
+import com.wynntils.mc.objects.CustomColor;
+import com.wynntils.mc.objects.Location;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.wynn.model.CompassModel;
+import java.util.List;
+import net.minecraft.client.Camera;
+import net.minecraft.world.phys.Vec3;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class WorldWaypointDistanceFeature extends UserFeature {
+
+    @Config
+    public CustomColor textColor = CommonColors.WHITE;
+
+    @Config
+    public int backgroundOpacity = 80;
```

Comment:
I would rather you make this a float (0-1 value). I think every other opacity config is a float.